cmd/submit.go

```diff
@@ -19,12 +19,14 @@ import (
 
 // submitCmd lets people upload a solution to the website.
 var submitCmd = &cobra.Command{
-	Use:     "submit FILE1 [FILE2 ...]",
+	Use: "submit [FILE1	 ...]",
 	Aliases: []string{"s"},
 	Short:   "Submit your solution to an exercise.",
 	Long: `Submit your solution to an Exercism exercise.
 
     Call the command with the list of files you want to submit.
+    If you omit the list of files, the CLI will attempt to submit
+    the appropriate solution files for the exercise.
```

Comment:
I think "attempt" sounds odd, like it might fail. Maybe:

```suggestion
    If you omit the list of files, the CLI will submit the
    default solution files for the exercise.
```

Reply:
It might fail, but for various reasons not under its control.  Full disk, lack of network connectivity, DDOS attack on Exercism, etc.  So there are points at which it may attempt to submit and fail.

Maybe if it fails the first time it automatically retries until it succeeds, and so success is guaranteed?